x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/_classification_exploration.scss

```diff
@@ -41,3 +41,9 @@ $labelColumnWidth: 80px;
     text-transform: none;
   }
 }
+
+.mlDataFrameAnalyticsClassification__evaluationMetrics {
+  width: 80%;
+  padding-left: 10%;
+  padding-right: 10%;
```

Comment:
why these styles? it doesn't seem like complying with the kibana guidelines 🤔 
also not sure about responsiveness, how does it look on smaller screens?

Reply:
Added these styles to prevent the table from taking up the full width since it will only have 3 columns and it's harder to read that way. Which section of the guidelines are you referring to specifically?

It will grow/shrink with the container as it's a percentage and here is a screen shot for smaller screens.

![image](https://user-images.githubusercontent.com/6446462/128934714-ab98d0b2-5092-4d54-9211-37bd8f31a1a6.png)

Agree with @cchaos's comment about cleaning up this view a but would prefer to do it in a separate PR as it's outside the scope of this PR and this view contains some workarounds for datagrid so it plays well with the flex layout and might need a bit of refactoring.